stores/noticeboard.js

```diff
@@ -85,7 +85,7 @@ export const useNoticeboardStore = defineStore({
       await this.fetch(id)
     },
     async comments(id, comments) {
-      await this.$api.noticeboard.action({
+      await api(this.config).noticeboard.action({
```

Comment:
Good spot!

Reply:
Yeah, took me a while to figure out why it wasn't working as expected :D